### PR TITLE
fixing bug in new rn validation

### PR DESCRIPTION
**Pull request by roysagi**
## Description
RN should not be checked in modified-files flow.
RN should be checked without structure validation in added-files flow

### CODE REVIEW

**Comment by Itay4** [comment]
https://app.circleci.com/pipelines/github/demisto/content/14034/workflows/52295d8e-646a-44f2-a253-886c876c1cf4/jobs/55528

File: demisto_sdk/commands/update_release_notes/update_rn.py [hunk 1]
```diff
@@ -78,12 +78,14 @@ def return_release_notes_path(self, input_version: str):
...
     def get_display_name(file_path):
-        struct = StructureValidator(file_path=file_path)
+        struct = StructureValidator(file_path=file_path, is_new_file=True)
         file_data = struct.load_data_from_file()
         if 'name' in file_data:
             name = file_data.get('name', None)
         elif 'TypeName' in file_data:
             name = file_data.get('TypeName', None)
+        elif 'brandName' in file_data:
+            name = file_data.get('TypeName', None)
```
Comment (ronykoz) [inline demisto_sdk/commands/update_release_notes/update_rn.py:88]: Guessing it should be:
```suggestion
            name = file_data.get('brandName', None)
```

File: demisto_sdk/commands/validate/file_validator.py [hunk 2]
```diff
@@ -509,6 +511,12 @@ def validate_added_files(self, added_files, file_type: str = None, modified_file
             if re.search(TEST_PLAYBOOK_REGEX, file_path, re.IGNORECASE) and not file_type:
                 continue
 
+            elif ('ReleaseNotes' in file_path) and not self.skip_pack_rn_validation:
```
Comment (ronykoz) [inline demisto_sdk/commands/validate/file_validator.py:514]: shouldnt it be releasenotes ?

Comment (roysagi) [reply demisto_sdk/commands/validate/file_validator.py:515]: @amshamah419 - I believe this is the unmodified file path so it should be RealseNotes - right?

Comment (amshamah419) [reply demisto_sdk/commands/validate/file_validator.py:515]: Correct. It should be `ReleaseNotes`.

**Comment by roysagi** [comment]
works when running on @Itay4 s branch using the ./demisto_sdk/utils/trigger_against_content_branch.sh script
https://circleci.com/gh/demisto/demisto-sdk/3962

Still fails but it seems the errors are legit.

File: demisto_sdk/commands/common/hook_validations/pack_unique_files.py [hunk 3]
```diff
@@ -41,6 +41,10 @@ def _add_error(self, error, file_path):
         """Adds error entry to a list under pack's name
         Returns True if added and false otherwise"""
         error_message, error_code = error
+
+        if '/' not in file_path:
```
Comment (ronykoz) [inline demisto_sdk/commands/common/hook_validations/pack_unique_files.py:45]: not sure that this is a good check, it will not work on windows

Comment (roysagi) [reply demisto_sdk/commands/common/hook_validations/pack_unique_files.py:45]: ok have another idea

**Comment by roysagi** [comment]
@ronykoz - circle fails on CL errors in validation - needs force merge

**Pull request merged**